src/MoBi.Presentation/MoBi.Presentation.csproj

```diff
@@ -26,13 +26,11 @@
 
   <ItemGroup>
     <PackageReference Include="Northwoods.GoWin" Version="5.2.0" />
+    <PackageReference Include="OSPSuite.Presentation.Importer" Version="10.0.92" />
```

Comment:
@msevestre  I ve been through the project files thoroughly to remove the duplicates and I find this line a bit weird...What is OSPSuite.Presentation.Importer and why its version is 10.0.92? Doesn't it come from Core? You think it is something deprecated?

Reply:
yes this does not exist anymore. This should be removed